### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by JaymzZh**

### CODE REVIEW

File: samples/PipeCI.Blob.Sample/Startup.cs [hunk 1]
```diff
@@ -39,14 +39,15 @@ public void Configure(IApplicationBuilder app, ILoggerFactory logger)
...
-</html>");
+                await context.Response.WriteAsync(@"
+                                                    <html>
+                                                        <body>
+                                                            <form action=""/blob/upload"" method=""post"" enctype=""multipart/form-data"">
+                                                                <input type=""file"" name=""file"" />
+                                                                <input type=""submit"" />
+                                                            </form>
+                                                        </body>
+                                                    </html>");
```
Comment (yukozh) [inline samples/PipeCI.Blob.Sample/Startup.cs]: It will be many spaces in browser.

Comment (JaymzZh) [inline samples/PipeCI.Blob.Sample/Startup.cs]: Is that matter? there are many spaces just in page source.

Comment (yukozh) [inline samples/PipeCI.Blob.Sample/Startup.cs]: See the things which Microsoft did. (https://github.com/aspnet/Localization/blob/dev/samples/LocalizationSample/Startup.cs#L79-L102)

Comment (JaymzZh) [inline samples/PipeCI.Blob.Sample/Startup.cs]: ~~Ok, I'll fix it.~~
Done.

**Pull request merged**